### PR TITLE
spline: more tests

### PR DESCRIPTION
**Pull request by d-netto**

### CODE REVIEW

File: test/spline_layer_test.jl [hunk 1]
```diff
@@ -32,14 +32,31 @@ function run_test(f, layer, atol)
     return sum(abs.(data_validate_pred.-data_validate_fn))/length(data_validate_fn) < atol
 end
 
-##test 01: affine function, Chebyshev and Polynomial basis
+ZygoteRules.@adjoint ZygoteRules.literal_getproperty(A::Tridiagonal, ::Val{:dl}) = A.dl,y -> Tridiagonal(dl,zeros(length(d)),zeros(length(du)),)
```
Comment (ChrisRackauckas) [inline test/spline_layer_test.jl:35]: put this in the library and we'll do a patch release when Zygote adds it.

Comment (d-netto) [reply test/spline_layer_test.jl]: Ok.

**Comment by d-netto** [comment]
TODO: add documentation and tutorial

**Comment by d-netto** [comment]
Some tests are stalling. Relevant tests for SplineLayer are passing.

**Pull request merged**